api/datadoghq/v2alpha1/datadogagent_default.go

```diff
@@ -465,21 +467,23 @@ func defaultFeaturesConfig(ddaSpec *DatadogAgentSpec) {
 		apiutils.DefaultBooleanIfUnset(&ddaSpec.Features.AdmissionController.Enabled, defaultAdmissionControllerEnabled)
 		apiutils.DefaultBooleanIfUnset(&ddaSpec.Features.AdmissionController.MutateUnlabelled, defaultAdmissionControllerMutateUnlabelled)
 		apiutils.DefaultStringIfUnset(&ddaSpec.Features.AdmissionController.ServiceName, defaultAdmissionServiceName)
-
 	}
-	agentSidecarInjection := ddaSpec.Features.AdmissionController.AgentSidecarInjection
-	if agentSidecarInjection != nil && agentSidecarInjection.Enabled != nil && *agentSidecarInjection.Enabled {
-		apiutils.DefaultBooleanIfUnset(&agentSidecarInjection.ClusterAgentCommunicationEnabled, defaultAdmissionControllerAgentSidecarClusterAgentEnabled)
+
+	// AdmissionControllerValidation Feature
+	if ddaSpec.Features.AdmissionController.AdmissionControllerValidationConfig == nil {
+		ddaSpec.Features.AdmissionController.AdmissionControllerValidationConfig = &AdmissionControllerValidationConfig{}
 	}
+	apiutils.DefaultBooleanIfUnset(&ddaSpec.Features.AdmissionController.AdmissionControllerValidationConfig.Enabled, defaultAdmissionControllerValidationEnabled)
 
-	// CWS Instrumentation in AdmissionController Feature
-	if ddaSpec.Features.AdmissionController.CWSInstrumentation == nil {
-		ddaSpec.Features.AdmissionController.CWSInstrumentation = &CWSInstrumentationConfig{}
```

Comment:
😬

Reply:
Ah actually it's the diff that is not clear. This code-block about CWS was duplicated right after this one, starting with line: https://github.com/DataDog/datadog-operator/pull/1461/files#diff-8c219cbd3c88a705dcbad675d025bcb767d00e3c2fe9f0916e5bd050e18bb456R489

Reply:
i got that! my emoji was about the accidental duplication that's been there a while